main/guides/getting-started/swaparoo-how-to-swap-assets-explainer.md

```diff
@@ -1,5 +1,106 @@
 # Swaparoo Contract
 
-## How to Swap Assets
+This smart contract is designed to allow two parties to swap assets between themselves, with a fee charged to one of the parties. The contract is started with a `feeAmount` and a `namesByAddressAdmin` object, which is used to retrieve a deposit facet for the second party.
```

Comment:
`namesByAddressAdmin` is actually excess authority; read-only access, i.e. `namesByAddress` should be enough. But it's not, due to a bug:

 - https://github.com/Agoric/agoric-sdk/issues/8113

the bug is fixed in master and due to hit mainnet in upgrade-14, I think. 

dunno how important it is to mention this.

Reply:
Added a note about it